### DISCUSSION TITLE
Which options are available for entries in dmenu mode?

### DISCUSSION THREAD

**Discussion by fdw** [davatorium/rofi#1729] [Q&A] [answered]
For example, I've found that you can specify a custom icon: https://github.com/davatorium/rofi/blob/next/releasenotes/1.5.3/release-1.5.3.markdown#icons-in-dmenu

The syntax allows for other options, too, but I haven't been able to find them documented anywhere. Did I miss it? Where can I see what is possible now?

**Comment by DaveDavenport** [comment, marked as answer]
Yes, it should be in the manpage(s).

**Reply by fdw** [reply]
Thanks, I found it in `rofi-script.5.markdown`, which is a bit surprising 😉

**Reply by DaveDavenport** [reply]
They share the same code and the `rofi-script` manpage its linked from `rofi-dmenu` manpage, so not that surprising.

**Reply by DaveDavenport** [reply]
https://github.com/davatorium/rofi/blob/next/doc/rofi-dmenu.5.markdown#dmenu-specific-commandline-flags

quote:

`A lot of these options can also be modified by the script using special input. See the rofi-script(5) manpage for more information about this syntax.`

**Reply by fdw** [reply]
Thanks for the link 🙂
However, I was looking not for the "normal" options, but for the [parsing row options](https://github.com/davatorium/rofi/blob/next/doc/rofi-script.5.markdown#parsing-row-options) (without knowing the name), which aren't explicitly mentioned in the `rofi-dmenu` manpage. But still, thanks to you I've found them 🙂

Would you like me to open a PR to add something to the `rofi-dmenu` manpage?

**Reply by DaveDavenport** [reply]
prs to improve documentation are always welcome.  please don't duplicate info (harder to maintain) but refer to it.
